bug-report when I did run flutter application

bug-report_1.md file prompted when I ran the flutter application:

Please file this bug report at https://github.com/flutter/flutter-intellij/issues/new.

---

## What happened

(please describe what you were doing when this exception occurred)

## Version information

Android Studio `Chipmunk | 2021.2.1 Patch 1` • Flutter plugin `io.flutter 69.0.2` • Dart plugin `212.5744`

Flutter 3.1.0-0.0.pre.1713 • channel master • https://github.com/flutter/flutter.git
Framework • revision 0527fd889e (7 days ago) • 2022-07-18 18:15:10 -0400
Engine • revision bb9938b1d7
Tools • Dart 2.19.0 (build 2.19.0-6.0.dev) • DevTools 2.15.0

## Exception

null

```
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated string at line 606 column 40 path $.packages[100].rootUri
	at com.google.gson.internal.Streams.parse(Streams.java:60)
	at com.google.gson.JsonParser.parseReader(JsonParser.java:85)
	at com.google.gson.JsonParser.parseReader(JsonParser.java:60)
	at com.google.gson.JsonParser.parseString(JsonParser.java:47)
	at io.flutter.utils.JsonUtils.parseString(JsonUtils.java:74)
	at io.flutter.sdk.FlutterPluginsLibraryManager.loadPackagesMap(FlutterPluginsLibraryManager.java:173)
	at io.flutter.sdk.FlutterPluginsLibraryManager.getFlutterPluginPaths(FlutterPluginsLibraryManager.java:143)
	at io.flutter.sdk.FlutterPluginsLibraryManager.updateFlutterPluginsImpl(FlutterPluginsLibraryManager.java:122)
	at io.flutter.sdk.FlutterPluginsLibraryManager.updateFlutterPlugins(FlutterPluginsLibraryManager.java:114)
	at com.intellij.openapi.project.DumbServiceImpl.lambda$smartInvokeLater$9(DumbServiceImpl.java:530)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:214)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:196)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:350)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:885)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:754)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:441)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:825)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:440)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:486)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: com.google.gson.stream.MalformedJsonException: Unterminated string at line 606 column 40 path $.packages[100].rootUri
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1564)
	at com.google.gson.stream.JsonReader.nextQuotedValue(JsonReader.java:1032)
	at com.google.gson.stream.JsonReader.nextString(JsonReader.java:816)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:702)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:723)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:715)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:723)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:698)
	at com.google.gson.internal.Streams.parse(Streams.java:48)
	... 38 more
```

--- a/flutter-idea/src/io/flutter/sdk/FlutterPluginsLibraryManager.java
+++ b/flutter-idea/src/io/flutter/sdk/FlutterPluginsLibraryManager.java
@@ -8,6 +8,7 @@ package io.flutter.sdk;
 import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
 import com.intellij.ProjectTopics;
 import com.intellij.openapi.application.ApplicationManager;
 import com.intellij.openapi.application.ModalityState;
@@ -190,7 +191,7 @@ public class FlutterPluginsLibraryManager extends AbstractLibraryManager<Flutter
         }
       }
     }
-    catch (IOException ignored) {
+    catch (IOException | JsonSyntaxException ignored) {
     }
     return result;
   }

